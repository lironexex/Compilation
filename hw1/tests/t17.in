// illegal strings 8 - unclosed string 3
"this line feed escape sequence is legal: \n"
"but the line feed char itself is illegal in a string:
"
// this won't be reached