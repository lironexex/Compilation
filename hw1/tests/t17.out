1 COMMENT 1
2 STRING this line feed escape sequence is legal: 

Error unclosed string
